src/x3d_operator_1d.f90

```diff
@@ -241,4 +308,631 @@ subroutine finalize(x3dop)
 
   end subroutine finalize
 
+   subroutine prepare(b, c, f, s, w, n)
```

Comment:
Would it be possible to comment what b,c,f,s,w,n are? I think they're part of the TDMA, even just a sketch of the routine would be nice

(I know this isn't the "Fortran" way :D)